notebooks/README.md

```diff
@@ -0,0 +1,42 @@
+# Trtion FIL backend with XGBoost
+
+This notebook is a reference for deploying a forest model trained using XGBoost library in Triton Inference Server with Forest Inference Library (FIL) backend. The notebook explains how one can deploy XGBoost model in Triton, check deployment status and send inference requests, set concurrent model execution and dynamic batching and find the best deployment configuration using Model Analyzer.
+
+## Requirements
+* Nvidia GPU (Pascal+ Recommended GPUs: T4, V100 or A100)
+* [Latest NVIDIA driver](https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html)
+* [Docker](https://docs.docker.com/get-docker/)
+* [The NVIDIA container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+
+## Run the Triton Inference Server container 
+
+**Note:** Please build the container locally inorder to use `model_analyzer` until the 21.08 release.
```

Comment:
```suggestion
**Note:** Due to a bug in release 21.07, Triton's `model_analyzer` cannot be used with the FIL backend until 21.08 is finalized. **Until then, you will need to build locally rather than pulling a pre-built container.** In the meantime, please follow [the instructions](https://github.com/triton-inference-server/fil_backend#building-locally) in the top-level README to build a development version of the 21.08 release.
```

Reply:
Also, please link to the relevant section of the README.